region/wreckedship/main.json

```diff
@@ -590,17 +590,16 @@
           "id": 2,
           "name": "Bottom Door",
           "nodeType": "door",
-          "nodeSubType": "grey",
+          "nodeSubType": "blue",
+          "lock": [
```

Comment:
This will probably trip up Mike's conversion help too, and the proposed format has evolved since. Can we take it out for now?

Reply:
I'm not quite sure how that found its way into the PR. Never meant to be pushed.